rasa/nlu/classifiers/DIET_classifier.py

```diff
@@ -570,7 +573,6 @@ def train(
             self.component_config[EVAL_NUM_EXAMPLES],
             self.component_config[EVAL_NUM_EPOCHS],
             batch_strategy=self.component_config[BATCH_STRATEGY],
```

Comment:
```suggestion
            self.component_config[BATCH_STRATEGY],
```
I think it is intuitive like this